test/functional/specs/desktop/C2588.js

```diff
@@ -1,21 +1,22 @@
 import { t } from "testcafe";
 
-fixture`C2588`.page(
-  "https://sandbox-qe.azurewebsites.net/sandbox/html/C2588.html"
-);
+const urlCollector = `http://127.0.0.1:8080/test/functional/sandbox/html/multiConfig.${
+  process.env.baseurl
+}.html`;
 
-const testDescription =
-  "Regression: Throw error when configure is executed multiple times.";
+fixture`C2588`.page(urlCollector);
 
 test.meta({
   ID: "C2588",
   SEVERITY: "P0",
   TEST_RUN: "Regression"
-})(testDescription, async () => {
-  const { error } = await t.getBrowserConsoleMessages();
+});
+
+test("Regression: Throw error when configure is executed multiple times.", async () => {
+  const message = await t.getBrowserConsoleMessages();
   await t
-    .expect(error)
-    .ok([
-      "[alloy] The library has already been configured and may only be configured once."
-    ]);
+    .expect(message.error[0])
```

Comment:
In general I try to be as lenient as possible when checking for error messages.  Changing an error message format slightly should not fail a test.  You could verify that there is one error message, or you could check to see that the error has the word 'already'.  Also could you verify that the second configure command returned a rejected promise?